typescript-express-reviews/.env.example

```diff
@@ -0,0 +1,4 @@
+STARGATE_JSON_API_URL=http://127.0.0.1:8080/v1/reviews
```

Comment:
are you supposed to source one of these environment files before running the example? The readme doesn't mention this. I got further after I did `source .env.example` before running `npm run seed`

Reply:
That's a very good point. The assumption was that you would create a `.env` file from `.env.example`, but those instructions weren't in the README. Also, `.env` currently doesn't have any sensitive credentials, so we'll just commit that to GitHub.

Reply:
I guess one reason to have `.env.example` (or `env.example`) is to alliw `.ignore` of real `.env` -- if everyone cloning repo is likely to modify it, avoid it being submitted as part of PR.